src/main/kotlin/no/nav/familie/tilbake/api/ForvaltningController.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.tilbake.api
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.tilbake.forvaltning.ForvaltningService
+import no.nav.familie.tilbake.sikkerhet.Behandlerrolle
+import no.nav.familie.tilbake.sikkerhet.Rolletilgangssjekk
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.math.BigInteger
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/forvaltning/")
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class ForvaltningController(private val forvaltningService: ForvaltningService) {
```

Comment:
Synes forvaltning blir litt utydelig. Det kan være veldig mye. Foreslår KravgrunnlagController

Reply:
Denne tjenesten kommer til å utvide for å implementere andre forvaltnings deler også. Disse tjenestene kommer vi ikke til å bruke fra frontend og kun brukes av forvaltningsteam via swagger. Derfor er det fornuftig til å skille den fra andre delene av tilbakekreving, tror jeg.